frontend-web/webclient/app/Project/Resources.tsx

```diff
@@ -178,6 +183,122 @@ const Resources: React.FunctionComponent = () => {
+                "" : `Earliest expiration: ${format(earliestExpiration, FORMAT)}`;
+
+            return <Accordion
+                icon={productTypeToIcon(key)}
+                title={prettierString(key)}
+                titleContent={<><Text color="text" mt="-4px" mr="16px">{expirationText}</Text><ResourceProgress value={Math.round(asPercent)} /></>}
+            >
+                <Border>
+                    <Spacer left={null} right={<><Text mt="-4px" mr="12px">Advanced view</Text><Toggle checked={advancedToggles.includes(key)} onChange={() => {
+                        if (advancedToggles.includes(key)) {
+                            setAdvancedToggles([...advancedToggles.filter(it => it !== key)]);
+                        } else {
+                            setAdvancedToggles([...advancedToggles, key]);
+                        }
+                    }} /></>} />
+                    <SimpleWalletView wallets={walletsList} advancedView={advancedToggles.includes(key)} />
+                </Border>
+            </Accordion>
+        })}
+    </AccordionWrapper>;
+}
+
+const Border = styled.div`
+    border-top: 1px solid lightGrey;
+    border-bottom: 1px solid lightGrey;
+    padding: 12px;
+`;
+
+function SimpleWalletView(props: {wallets: Wallet[]; advancedView: boolean;}): JSX.Element {
+    return <SimpleWalletRowWrapper>
+        {props.wallets.map(wallet => {
+            const asPercent = resultAsPercent(totalUsageFromWallet(wallet));
+            const expiration = wallet.allocations.reduce((lowest, wallet) =>
+                wallet.endDate && wallet.endDate < lowest ? wallet.endDate! : lowest, VERY_HIGH_DATE_VALUE
+            );
+            const expirationText = expiration === VERY_HIGH_DATE_VALUE ? "" : `Earliest expiration: ${format(expiration, FORMAT)}`;
+            return (
+                <SimpleAllocationRowWrapper>
+                    <Spacer
+                        px="30px"
+                        left={<Text color="text" mt="-4px">{wallet.paysFor.name} @ {wallet.paysFor.provider}</Text>}
+                        right={<><Text color="text" mt="-4px" mr="16px">{expirationText}</Text><ResourceProgress value={Math.round(asPercent)} /></>}
+                    />
+                    {props.advancedView ? <VisualizationSection><WalletViewer wallet={wallet} /></VisualizationSection> : null}
+                </SimpleAllocationRowWrapper>
+            );
+        })}
+    </SimpleWalletRowWrapper>;
+}
+
+const SimpleAllocationRowWrapper = styled.div``;
+const SimpleWalletRowWrapper = styled.div`
+    & > ${SimpleAllocationRowWrapper}:not(:last-child) {
+        vertical-align: center;
+        border-bottom: 1px solid #d3d3d3;
+    }
+    
+    & > ${SimpleAllocationRowWrapper} {
+        margin-top: 12px;
+        padding-bottom: 10px;
+        border-bottom: 0px solid black;
+    }
+`;
+
+function totalUsageFromMultipleWallets(wallets: Wallet[]): {balance: number, initialBalance: number} {
```

Comment:
It would probably make sense to use a type since we are returning it from multiple functions.